integration/e2e/api/query_range_test.go

```diff
@@ -55,17 +55,55 @@ sendLoop:
 	for _, query := range []string{
 		"{} | rate()",
 		"{} | compare({status=error})",
+		"{} | count_over_time()",
+		"{} | min_over_time(duration)",
+		"{} | max_over_time(duration)",
+		"{} | avg_over_time(duration)",
+		"{} | sum_over_time(duration)",
+		"{} | quantile_over_time(duration, .5)",
+		"{} | quantile_over_time(duration, .5, 0.9, 0.99)",
+		"{} | histogram_over_time(duration)",
+		"{} | count_over_time() by (status)",
+		"{status != error} | count_over_time() by (status)",
 	} {
 		t.Run(query, func(t *testing.T) {
-			callQueryRange(t, tempo.Endpoint(tempoPort), query, debugMode)
+			queryRangeRes := callQueryRange(t, tempo.Endpoint(tempoPort), query, debugMode)
```

Comment:
I think this test should be rewritten to a deterministic table-driven test asserting that the data returned is correct:

- MakeThriftBatch generates random data; this should be changed to something we can control
- We should ensure we have traces in the backend and in recent data to test the different aggregation layers
- We should test a comprehensive list of metrics with and without aggregation and check the correctnes of the returned series